Doc/extending/windows.rst

```diff
@@ -34,12 +34,11 @@ A Cookbook Approach
 ===================
 
 There are two approaches to building extension modules on Windows, just as there
-are on Unix: use the :mod:`distutils` package to control the build process, or
-do things manually.  The distutils approach works well for most extensions;
-documentation on using :mod:`distutils` to build and package extension modules
-is available in :ref:`distutils-index`.  If you find you really need to do
-things manually, it may be instructive to study the project file for the
-:source:`winsound <PCbuild/winsound.vcxproj>` standard library module.
+are on Unix: use the ``setuptools`` package to control the build process, or
```

Comment:
Missing ``:mod:`` ??

Reply:
Not sure. Because setuptools is not actually part of the stdlib.

Reply:
Using `:mod:` is correct, since it produces the correct formatting for module names; however, use `!` to avoid producing an actual link.

```suggestion
are on Unix: use the :mod:`!setuptools` package to control the build process, or
```